mava/environment_loop.py

```diff
@@ -585,36 +580,24 @@ def should_run_loop(eval_interval_condition: Tuple) -> bool:
 
         episode_count, step_count = 0, 0
```

Comment:
We can also try removing these now.

Reply:
And then we can make the while loop be a `while True` loop since we don't handle termination at the environment loop level anymore but in the termination component.

Reply:
oh I see, I will check the single process first and we can confirm from there!